hertzbeat-manager/src/main/resources/define/app-hikvision_isapi.yml

```diff
@@ -97,32 +97,36 @@ metrics:
         i18n:
           zh-CN: 设备名称
           en-US: Device Name
-        xpath: deviceName
```

Comment:
hi, can the the special key `locator`(only for xml) use the `alaiaFields` and `calculates` to instead ?